docs/content/Caching/Using-Pre-Aggregations.md

```diff
@@ -245,16 +255,71 @@ slow to return results.
 (such as AWS Athena and BigQuery). Repeatedly querying for this data can easily
 rack up costs.
 
+## Optimizing Pre-Aggregation Build Times
+
+<!-- prettier-ignore-start -->
+[[info | ]]
+| For ideal performance, pre-aggregations should be built using a dedicated
+| Refresh Worker. [See here for more details][ref-prod-list-refresh].
+<!-- prettier-ignore-end -->
+
+By default, Cube.js will use the source database as a temporary staging area for
```

Comment:
@ovr @hassankhan Does it apply to batching as well?

Reply:
We are using tables for not a read-only mode for all strategies: download/streaming/export